cg/store/api/status.py

```diff
@@ -233,7 +233,7 @@ def get_cases_to_compress(self, date_threshold: datetime) -> List[Family]:
         """Return all cases that are ready to be compressed by SPRING."""
```

Comment:
```suggestion
        """Return all cases that are ready to be compressed by SPRING created before a given date."""
```
Also maybe change the parameter name from `date_threshold` to `before_date` or something more meaningful